lib/ansible/plugins/connection/podman.py

```diff
@@ -79,7 +99,14 @@ def _podman(self, cmd, cmd_args=None, in_data=None, use_container_id=True):
         :param in_data: data passed to podman's stdin
         :return: return code, stdout, stderr
         """
-        local_cmd = ['podman', cmd]
+        podman_exec = self.get_option('podman_executable')
+        podman_cmd = distutils.spawn.find_executable(podman_exec)
+        if not podman_cmd:
+            raise AnsibleError("%s command not found in PATH" % podman_exec)
+        local_cmd = [podman_cmd]
+        if self.get_option('podman_extra_args'):
+            local_cmd += self.get_option('podman_extra_args').split()
```

Comment:
How about not using `.split()`, but `shlex.split()` ([reference](https://docs.python.org/3/library/shlex.html))? (I wish the docker connection plugin would use that one as well, but changing it is not that easy due to backwards compatibility...)

Reply:
yeah, totally reasonable